build/Dockerfile

```diff
@@ -1,45 +1,270 @@
 ARG GOLANG_CONTAINER=golang:latest
+ARG NGINX_VERSION
+ARG BUILD_OS
 
-FROM nginx:1.19.3 AS base
+######### Base image for Debian #########
+FROM nginx:$NGINX_VERSION AS debian
+ARG IC_VERSION
```

Comment:
Would it still work if this `ARG` was defined at the top with the others?

Reply:
It wasn't working for some reason...I can double-check

Reply:
No problem at all.